sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -280,13 +281,15 @@ export class ApplicationStudentsController extends BaseController {
     const programYear = await this.programYearService.getActiveProgramYear(
       payload.programYearId,
     );
+    const { offeringIntensity } =
```

Comment:
I did a refactor in my current PR that will cause some adaptations in this code. I will handle it right after this PR is merged.